gae_dashboard/dos_alert.py

```diff
@@ -283,10 +287,18 @@ def dos_detect(end):
     for ip, rows in ip_groups:
         alerted_ip = False
         for row in rows:
-            to_alert = not any([
-                re.match(filter_regex, row['url'])
-                for filter_regex in DOS_SAFELIST_URL_REGEX
-            ])
+            # If a matching url is found, check for adjusted/overriden max
+            # requests for that particular route
+            # TODO(drosile): maybe refactor this to be cleaner?
+            alerting_url_regexes = [
+                    filter_regex for filter_regex
+                    in DOS_SAFELIST_URL_REGEX.keys()
+                    if re.match(filter_regex, row['url'])]
+            if alerting_url_regexes:
+                regex = alerting_url_regexes[1]
```

Comment:
Do we mean first element?  We might also comment on that we only use other multiplier from first match.
```suggestion
                regex = alerting_url_regexes[0]
```